client/blocks/image-editor/index.jsx

```diff
@@ -71,22 +71,18 @@ class ImageEditor extends Component {
 
 	editCanvasRef = createRef();
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( newProps ) {
+	componentDidUpdate( prevProps ) {
 		const { media: currentMedia } = this.props;
```

Comment:
Did you mean to alter `this.props` to `prevProps` here?

Reply:
We used to destructure from props before the update but in the next line you can see we're switching usages so this should be fine.